tools.go

```diff
@@ -0,0 +1,12 @@
+//go:build tools
```

Comment:
were you going to remove these build flags?

Reply:
replied in relevant thread